src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -19,6 +20,21 @@ export async function getTxCounterparties (transactionList) {
   return noDuplicatesAgentList
 }
 
+// // PLACEHOLDER for presenting filtered host earnings :
```

Comment:
Let's remove all the commented code in this file. I'm not confident this is how the earnings stuff will work, and having it in here is debt.